index.js

```diff
@@ -8,6 +8,7 @@ const strictValueDefaults = ['inherit', 'initial', 'unset'];
 
 // See https://stylelint.io/user-guide/configure
 module.exports = {
+    "customSyntax": "postcss-styled",
```

Comment:
I think only the consuming repo would know this, right? previously `stylelint` was un-opinionated about which build tooling you were using. 

can we not put `customSyntax` in `Infinity` config as an override?

Reply:
sorry just saw this, so firstly this is now reverted back to `postcss-scss` as `postcss-styled` was the wrong choice. And I believe for us we can just set the `customSyntax` here in this main config and not provide it again in repos individually. If I have understood [customSyntax](https://stylelint.io/user-guide/usage/options#customsyntax) well!